src/Encamina.Enmarcha.SemanticKernel/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -15,26 +15,42 @@ namespace Microsoft.Extensions.DependencyInjection;
 public static class IServiceCollectionExtensions
 {
     /// <summary>
-    /// Adds and configures the <see cref="MemoryManager"/> type as transient service instance of the <see cref="IMemoryManager"/> service to the <see cref="IServiceCollection"/>.
+    /// Adds and configures the <see cref="MemoryStoreExtender"/> type as transient service instance of the <see cref="IMemoryStoreExtender"/> service to the <see cref="IServiceCollection"/>.
     /// </summary>
     /// <remarks>
-    /// This method registers the <see cref="MemoryManager"/> type as a transient service due to its dependency on <see cref="Kernel"/>, which is typically registered as transient or scoped (rarely as singleton).
+    /// This method registers the <see cref="MemoryStoreExtender"/> type as a transient service due to its dependency on <see cref="Kernel"/>, which is typically registered as transient or scoped (rarely as singleton).
     /// </remarks>
     /// <param name="services">The <see cref="IServiceCollection"/> to add services to.</param>
     /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+    [Obsolete("This extension is obsolete and will be removed in a future version. Use the Microsoft.Extensions.DependencyInjection.IServiceCollectionExtensions.AddMemoryStoreExtender() extension instead.", false)]
     public static IServiceCollection AddMemoryManager(this IServiceCollection services)
     {
         services.TryAddSingleton<IMemoryManager, MemoryManager>();
 
         return services;
     }
 
+    /// <summary>
+    /// Adds and configures the <see cref="MemoryStoreExtender"/> type as transient service instance of the <see cref="IMemoryStoreExtender"/> service to the <see cref="IServiceCollection"/>.
```

Comment:
It says that we are adding the MemoryStoreExtender as transient yet in the code is being added as singleton, is the comment or the code wrong?

Reply:
Thank you for your contribution. I will update it.